### PR TITLE
added homeWork_6

### PR DESCRIPTION
**Pull request by lesiaturuta**
Acceptance Criteria for 1

- [ ]  Mentor is added to pr
- [ ]  AC copied to pr description
- [ ]  Link to deployed version is added to pr
- [ ]  LayoutComponent with a static header and footer, and an Outlet for dynamic content, is added.
- [ ]  Two routes, AboutPage and ProductsPage, are added.
- [ ]  The AboutPage is set as the index route.
- [ ]  A route for invalid pages is added, displaying Page not found with a link to the AboutPage.

Acceptance Criteria for 1*

- [ ]  The layout for ProductPage is added according to the design.
- [ ]  An additional route for ProductPage with dynamic parameters :id is added.
- [ ]  An additional request getProduct is implemented, using the product's id.

link: https://lesiaturuta.github.io/fe-react-2024/

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -141,26 +142,31 @@ const App = () => {
...
-                        increaseCounter={increaseCounter}
-                        decrementCounter={decrementCounter}
-                        getProductsByCategory={getProductsByCategory}
-                        getSortByName={getSortByName}
-                    />
-                )}
-                <FooterComponent theme={theme} />
+                <Routes>
+                    <Route path={'/'} element={<LayoutComponent theme={theme} amountCart={amountCart} changeTheme={changeTheme} />}>
+                        <Route path={'/'} index element={<About theme={theme} />} />
```
Comment (vadym-zinchenko-moc) [inline src/App.tsx]: Do we need path here?

File: src/App.tsx [hunk 2]
```diff
@@ -141,26 +142,31 @@ const App = () => {
...
+                                    increasePage={increasePage}
+                                    decrementPage={decrementPage}
+                                    increaseCounter={increaseCounter}
+                                    decrementCounter={decrementCounter}
+                                    getProductsByCategory={getProductsByCategory}
+                                    getSortByName={getSortByName}
+                                />
+                            }
+                        />
+                        <Route path={'/product/:id'} element={<ProductPageComponent theme={theme} />} />{' '}
```
Comment (vadym-zinchenko-moc) [inline src/App.tsx]: path={'product/:id'} is it work without `/`?

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (lesiaturuta) [reply src/App.tsx]: fixed

File: src/App.tsx (hunk 2 above)
Comment (lesiaturuta) [reply src/App.tsx]: fixed

**Review by vadym-zinchenko-moc — APPROVED** [review]

**Pull request merged**